sdk/java-sdk/sdk/spec/proto/runtime/v1/runtime.proto

```diff
@@ -0,0 +1,545 @@
+syntax = "proto3";
```

Comment:
why do we duplicate a runtime.proto here?

Reply:
removed.
I duplicated it here because I wanted to make java-sdk project independent with Layotto project. 
Since we won't split it into two projects,I removed the redundent proto files